src/sentry/api/endpoints/organization_releases.py

```diff
@@ -217,6 +217,14 @@ class OrganizationReleasesEndpoint(
         ]
     )
 
+    def get_projects(self, request: Request, organization, project_ids=None):
+        return super().get_projects(
+            request,
+            organization,
+            project_ids=project_ids,
+            include_all_accessible=False if "GET" == request.method else True,
+        )
```

Comment:
Kinda hacky here ... maybe there's a better way?

Reply:
Looking at the comment [here](https://github.com/getsentry/sentry/blob/f1decaa1983a8ee909221a885c991e0da8532538/src/sentry/api/bases/organization.py#L187-L189), it looks like there's some bigger refactoring work to do to remove the parameter altogether. Could probably clean this up along with the refactor if needed.